### ISSUE TITLE
Can't install on 16.04.1 LTS due to PHP5 missing. 

### ISSUE THREAD

**Issue by changeableface** [olejon/spotcommander#105]
Using the latest install .deb from here: http://www.olejon.net/code/spotcommander/files/spotcommander-latest.deb I am unable to install on Ubuntu 16.04.1 LTS.

Can't see anywhere mentioned that SpotCommander isn't support on 16.04, but this looks to be an issue with not being able to install php5 on 16.04 - any fix for this? 

```
base@music-server:~$ sudo dpkg -i spotcommander-latest.deb
Selecting previously unselected package spotcommander.
(Reading database ... 254695 files and directories currently installed.)
Preparing to unpack spotcommander-latest.deb ...
Unpacking spotcommander (12.4-1) ...
dpkg: dependency problems prevent configuration of spotcommander:
 spotcommander depends on libapache2-mod-php5; however:
  Package libapache2-mod-php5 is not installed.
 spotcommander depends on php5-cli; however:
  Package php5-cli is not installed.
 spotcommander depends on php5-sqlite; however:
  Package php5-sqlite is not installed.
 spotcommander depends on php5-curl; however:
  Package php5-curl is not installed.
 spotcommander depends on php5-json; however:
  Package php5-json is not installed.

dpkg: error processing package spotcommander (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 spotcommander
```

**Comment by olejon** [comment]
Use GDebi or Ubuntu Software Center instead to install SpotCommander. It will take care of the dependencies. dpkg -i alone doesn't do that.

**Comment by changeableface** [comment]
Hi! 

Thanks for the super swift reply! 

When installing with Software centre I get "Dependency is not satisfiable: libapache2-mod-php5"

:(

- Matt

**Comment by olejon** [comment]
The problem was that they changed so many package names in 16.04, and to make the deb compatible with all versions, I had to add optional package names for the dependency resolver. This isn't something difficult, and worked fine the many times I tried it on 16.04 in a virtual machine. I even tried 32-bit if I recall correctly. Have you tried GDebi?

**Comment by olejon** [comment]
We'll, I'll try on a fresh VM when I have some time. If someone else have this problem, please confirm it.

**Comment by changeableface** [comment]
Hi Olejon, 

Appreciate any help you can provide otherwise we're rocking solo with no music for a while. 

- Matt

**Comment by olejon** [comment]
Just installed a fresh Ubuntu 16.04.1 ISO in VirtualBox and did what I always do after a fresh install:

1. `sudo apt-get update && sudo apt-get dist-upgrade`
2. Reboot
3. Install VirtualBox Guest Additions (*just necessary for the VM*)
4. Reboot (*just necessary for the VM because of Guest Additions*)
5. Go through System Settings (*just my way to set up a new VM to my preference*)
5. Install Spotify and SpotCommander as in the [installation instructions](http://www.olejon.net/code/spotcommander/?install)

It works just fine. Watch this video: https://dl.dropboxusercontent.com/u/8593574/Diverse/Issue105.mp4

As you can see, I install libgcrypt first, as in the installation instructions, then the Spotify Client (recommended version), then SpotCommander. Then of course I did the usual things, open Spotify, login and adjust the preferences. This doesn't have anything to do with installing SpotCommander though. Then I logged out and back in again as according to the SpotCommander installation instructions, and everything **worked**.

**Comment by olejon** [comment]
BTW: I have updated the comment above several times just so you know if you have already read it, read it again.

**Comment by changeableface** [comment]
Hi, 

The setup was running 14.04.01 LTS (completely standard on a standalone rack server, so not a VM) with Spotify and Spotcommander all working correctly. 

We preformed an upgrade to 16.04.01 LTS and Spotcommander would not start. I removed and attempted to reinstall assuming some package was missing or broke and then I came across the issues listed above. 

Spotify continues to run correctly, just the commander that's proving tricky. 

Does that help?

**Comment by olejon** [comment]
Hm, must be the upgrade then. I have never upgraded Ubuntu as I don't really trust their upgrade mechanism to work correctly, and you run into stuff like this.

In 16.04 they changed the package name from `libapache2-mod-php5` to `libapache2-mod-php`. What happens if you run: `sudo apt-get install libapache2-mod-php` ?

**Comment by changeableface** [comment]
```
dpkg: dependency problems prevent configuration of spotcommander:
 spotcommander depends on libapache2-mod-php5; however:
  Package libapache2-mod-php5 is not installed.
 spotcommander depends on php5-cli; however:
  Package php5-cli is not installed.
 spotcommander depends on php5-sqlite; however:
  Package php5-sqlite is not installed.
 spotcommander depends on php5-curl; however:
  Package php5-curl is not installed.
 spotcommander depends on php5-json; however:
  Package php5-json is not installed.
```

 

Thanks for your help, given I have nothing vital to lose I'll save us both time and just nuke the server and start again. It's not ideal, but it will only take an hour or so on and off.

**Comment by olejon** [comment]
Was that what you got when trying to install `libapache2-mod-php` ? In that case, since you first installed SpotCommander using dpkg, purge SpotCommander with `sudo apt-get purge spotcommander`. Then try again: `sudo apt-get install libapache2-mod-php` .

**Comment by changeableface** [comment]
```
base@music-server:~$ sudo apt-get install libapache2-mod-php
[sudo] password for base:
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  libapache2-mod-php7.0 php-common php7.0-cli php7.0-common php7.0-json
  php7.0-opcache php7.0-readline
Suggested packages:
  php-pear
The following NEW packages will be installed
  libapache2-mod-php libapache2-mod-php7.0 php-common php7.0-cli php7.0-common
  php7.0-json php7.0-opcache php7.0-readline
0 to upgrade, 8 to newly install, 0 to remove and 1 not to upgrade.
Need to get 3,475 kB/3,488 kB of archives.
After this operation, 13.9 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-common amd64 7.0.9-1+deb.sury.org~xenial+1 [861 kB]
Get:2 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-json amd64 7.0.9-1+deb.sury.org~xenial+1 [16.8 kB]
Get:3 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-opcache amd64 7.0.9-1+deb.sury.org~xenial+1 [76.0 kB]
Get:4 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-readline amd64 7.0.9-1+deb.sury.org~xenial+1 [12.8 kB]
Get:5 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-cli amd64 7.0.9-1+deb.sury.org~xenial+1 [1,281 kB]
Get:6 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 libapache2-mod-php7.0 amd64 7.0.9-1+deb.sury.org~xenial+1 [1,223 kB]
Get:7 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 libapache2-mod-php all 1:7.0+43+deb.sury.org~xenial+1 [4,816 B]
Fetched 3,475 kB in 0s (11.6 MB/s)
Selecting previously unselected package php-common.
(Reading database ... 254700 files and directories currently installed.)
Preparing to unpack .../php-common_1%3a43+deb.sury.org~xenial+1_all.deb ...
Unpacking php-common (1:43+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-common.
Preparing to unpack .../php7.0-common_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-common (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-json.
Preparing to unpack .../php7.0-json_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-json (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-opcache.
Preparing to unpack .../php7.0-opcache_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-opcache (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-readline.
Preparing to unpack .../php7.0-readline_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-readline (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-cli.
Preparing to unpack .../php7.0-cli_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-cli (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package libapache2-mod-php7.0.
Preparing to unpack .../libapache2-mod-php7.0_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking libapache2-mod-php7.0 (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package libapache2-mod-php.
Preparing to unpack .../libapache2-mod-php_1%3a7.0+43+deb.sury.org~xenial+1_all.deb ...
Unpacking libapache2-mod-php (1:7.0+43+deb.sury.org~xenial+1) ...
Processing triggers for man-db (2.7.5-1) ...
Setting up php-common (1:43+deb.sury.org~xenial+1) ...
Setting up php7.0-common (7.0.9-1+deb.sury.org~xenial+1) ...
Setting up php7.0-json (7.0.9-1+deb.sury.org~xenial+1) ...
Setting up php7.0-opcache (7.0.9-1+deb.sury.org~xenial+1) ...
Setting up php7.0-readline (7.0.9-1+deb.sury.org~xenial+1) ...
Setting up php7.0-cli (7.0.9-1+deb.sury.org~xenial+1) ...
update-alternatives: using /usr/bin/php7.0 to provide /usr/bin/php (php) in auto mode
update-alternatives: using /usr/bin/phar7.0 to provide /usr/bin/phar (phar) in auto mode
update-alternatives: using /usr/bin/phar.phar7.0 to provide /usr/bin/phar.phar (phar.phar) in auto mode
Replacing config file /etc/php/7.0/cli/php.ini with new version
Setting up libapache2-mod-php7.0 (7.0.9-1+deb.sury.org~xenial+1) ...
Replacing config file /etc/php/7.0/apache2/php.ini with new version
apache2_invoke: Enable module php7.0
Setting up libapache2-mod-php (1:7.0+43+deb.sury.org~xenial+1) ...
base@music-server:~$ sudo dpkg -i spotcommander-latest.deb
Selecting previously unselected package spotcommander.
(Reading database ... 254824 files and directories currently installed.)
Preparing to unpack spotcommander-latest.deb ...
Unpacking spotcommander (12.4-1) ...
dpkg: dependency problems prevent configuration of spotcommander:
 spotcommander depends on libapache2-mod-php5; however:
  Package libapache2-mod-php5 is not installed.
 spotcommander depends on php5-cli; however:
  Package php5-cli is not installed.
 spotcommander depends on php5-sqlite; however:
  Package php5-sqlite is not installed.
 spotcommander depends on php5-curl; however:
  Package php5-curl is not installed.
 spotcommander depends on php5-json; however:
  Package php5-json is not installed.
 spotcommander depends on inotify-tools; however:
  Package inotify-tools is not installed.
 spotcommander depends on xdotool; however:
  Package xdotool is not installed.

dpkg: error processing package spotcommander (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 spotcommander
base@music-server:~$ sudo apt-get install -f
Reading package lists... Done
Building dependency tree
Reading state information... Done
Correcting dependencies... Done
The following packages were automatically installed and are no longer required:
  inotify-tools libinotifytools0 libxdo3 xdotool
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  inotify-tools libinotifytools0 libxdo3 xdotool
The following packages will be REMOVED
  spotcommander
The following NEW packages will be installed
  inotify-tools libinotifytools0 libxdo3 xdotool
0 to upgrade, 4 to newly install, 1 to remove and 1 not to upgrade.
1 not fully installed or removed.
Need to get 0 B/99.1 kB of archives.
After this operation, 4,882 kB disk space will be freed.
Do you want to continue? [Y/n] y
(Reading database ... 255207 files and directories currently installed.)
Removing spotcommander (12.4-1) ...
Selecting previously unselected package libinotifytools0.
(Reading database ... 254824 files and directories currently installed.)
Preparing to unpack .../libinotifytools0_3.14-1ubuntu1_amd64.deb ...
Unpacking libinotifytools0 (3.14-1ubuntu1) ...
Selecting previously unselected package inotify-tools.
Preparing to unpack .../inotify-tools_3.14-1ubuntu1_amd64.deb ...
Unpacking inotify-tools (3.14-1ubuntu1) ...
Selecting previously unselected package libxdo3:amd64.
Preparing to unpack .../libxdo3_1%3a3.20150503.1-2_amd64.deb ...
Unpacking libxdo3:amd64 (1:3.20150503.1-2) ...
Selecting previously unselected package xdotool.
Preparing to unpack .../xdotool_1%3a3.20150503.1-2_amd64.deb ...
Unpacking xdotool (1:3.20150503.1-2) ...
Processing triggers for man-db (2.7.5-1) ...
Setting up libinotifytools0 (3.14-1ubuntu1) ...
Setting up inotify-tools (3.14-1ubuntu1) ...
Setting up libxdo3:amd64 (1:3.20150503.1-2) ...
Setting up xdotool (1:3.20150503.1-2) ...
Processing triggers for libc-bin (2.23-0ubuntu3) ...
base@music-server:~$ sudo dpkg -i spotcommander-latest.deb
Selecting previously unselected package spotcommander.
(Reading database ... 254850 files and directories currently installed.)
Preparing to unpack spotcommander-latest.deb ...
Unpacking spotcommander (12.4-1) ...
dpkg: dependency problems prevent configuration of spotcommander:
 spotcommander depends on libapache2-mod-php5; however:
  Package libapache2-mod-php5 is not installed.
 spotcommander depends on php5-cli; however:
  Package php5-cli is not installed.
 spotcommander depends on php5-sqlite; however:
  Package php5-sqlite is not installed.
 spotcommander depends on php5-curl; however:
  Package php5-curl is not installed.
 spotcommander depends on php5-json; however:
  Package php5-json is not installed.

dpkg: error processing package spotcommander (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 spotcommander
base@music-server:~$
```

**Comment by olejon** [comment]
Was that what you got when trying to install `libapache2-mod-php` ? In that case, since you first installed SpotCommander using dpkg, purge SpotCommander with `sudo apt-get purge spotcommander`. Then try again: `sudo apt-get install libapache2-mod-php` . Pretty sure this can be solved, but probably has to do with the upgrading leaving legacy stuff behind. Probably you would see this in `/etc/apt/sources.list` and the files in `/etc/apt/sources.list.d/`. An upgrade would probably avoid upgrading server stuff like Apache and PHP in this case...

**Comment by olejon** [comment]
So the dependencies for SpotCommander on 16.04 are:

`sudo apt-get install apache2 apache2-utils inotify-tools libapache2-mod-php php-cli php-curl php-json php-sqlite3 qdbus xautomation xdotool`. Can you install those without problems?

**Comment by changeableface** [comment]
Yup all those are install with no errors. 

```
base@music-server:~$ sudo apt-get install apache2 apache2-utils inotify-tools libapache2-mod-php php-cli php-curl php-json php-sqlite3 qdbus xautomation xdotool
Reading package lists... Done
Building dependency tree
Reading state information... Done
qdbus is already the newest version (4:4.8.7+dfsg-5ubuntu2).
inotify-tools is already the newest version (3.14-1ubuntu1).
inotify-tools set to manually installed.
xdotool is already the newest version (1:3.20150503.1-2).
xdotool set to manually installed.
apache2 is already the newest version (2.4.18-2ubuntu3.1).
apache2-utils is already the newest version (2.4.18-2ubuntu3.1).
libapache2-mod-php is already the newest version (1:7.0+43+deb.sury.org~xenial+1).
The following additional packages will be installed:
  php7.0-curl php7.0-sqlite3
The following NEW packages will be installed
  php-cli php-curl php-json php-sqlite3 php7.0-curl php7.0-sqlite3 xautomation
0 to upgrade, 7 to newly install, 0 to remove and 1 not to upgrade.
Need to get 70.9 kB/108 kB of archives.
After this operation, 486 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php-cli all 1:7.0+43+deb.sury.org~xenial+1 [4,714 B]
Get:2 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-curl amd64 7.0.9-1+deb.sury.org~xenial+1 [27.6 kB]
Get:3 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php-curl all 1:7.0+43+deb.sury.org~xenial+1 [4,660 B]
Get:4 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php-json all 1:7.0+43+deb.sury.org~xenial+1 [4,660 B]
Get:5 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.0-sqlite3 amd64 7.0.9-1+deb.sury.org~xenial+1 [24.6 kB]
Get:6 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php-sqlite3 all 1:7.0+43+deb.sury.org~xenial+1 [4,668 B]
Fetched 70.9 kB in 0s (955 kB/s)
Selecting previously unselected package php-cli.
(Reading database ... 254850 files and directories currently installed.)
Preparing to unpack .../php-cli_1%3a7.0+43+deb.sury.org~xenial+1_all.deb ...
Unpacking php-cli (1:7.0+43+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-curl.
Preparing to unpack .../php7.0-curl_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-curl (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package php-curl.
Preparing to unpack .../php-curl_1%3a7.0+43+deb.sury.org~xenial+1_all.deb ...
Unpacking php-curl (1:7.0+43+deb.sury.org~xenial+1) ...
Selecting previously unselected package php-json.
Preparing to unpack .../php-json_1%3a7.0+43+deb.sury.org~xenial+1_all.deb ...
Unpacking php-json (1:7.0+43+deb.sury.org~xenial+1) ...
Selecting previously unselected package php7.0-sqlite3.
Preparing to unpack .../php7.0-sqlite3_7.0.9-1+deb.sury.org~xenial+1_amd64.deb ...
Unpacking php7.0-sqlite3 (7.0.9-1+deb.sury.org~xenial+1) ...
Selecting previously unselected package php-sqlite3.
Preparing to unpack .../php-sqlite3_1%3a7.0+43+deb.sury.org~xenial+1_all.deb ...
Unpacking php-sqlite3 (1:7.0+43+deb.sury.org~xenial+1) ...
Selecting previously unselected package xautomation.
Preparing to unpack .../xautomation_1.09-1_amd64.deb ...
Unpacking xautomation (1.09-1) ...
Processing triggers for libapache2-mod-php7.0 (7.0.9-1+deb.sury.org~xenial+1) ...
Processing triggers for man-db (2.7.5-1) ...
Setting up php-cli (1:7.0+43+deb.sury.org~xenial+1) ...
Setting up php7.0-curl (7.0.9-1+deb.sury.org~xenial+1) ...

Creating config file /etc/php/7.0/mods-available/curl.ini with new version
Setting up php-curl (1:7.0+43+deb.sury.org~xenial+1) ...
Setting up php-json (1:7.0+43+deb.sury.org~xenial+1) ...
Setting up php7.0-sqlite3 (7.0.9-1+deb.sury.org~xenial+1) ...

Creating config file /etc/php/7.0/mods-available/sqlite3.ini with new version

Creating config file /etc/php/7.0/mods-available/pdo_sqlite.ini with new version
Setting up php-sqlite3 (1:7.0+43+deb.sury.org~xenial+1) ...
Setting up xautomation (1.09-1) ...
Processing triggers for libapache2-mod-php7.0 (7.0.9-1+deb.sury.org~xenial+1) ...
base@music-server:~$
```

Still can't install spot commander. 

```
base@music-server:~$ sudo dpkg -i spotcommander-latest.deb
Selecting previously unselected package spotcommander.
(Reading database ... 254900 files and directories currently installed.)
Preparing to unpack spotcommander-latest.deb ...
Unpacking spotcommander (12.4-1) ...
dpkg: dependency problems prevent configuration of spotcommander:
 spotcommander depends on libapache2-mod-php5; however:
  Package libapache2-mod-php5 is not installed.
 spotcommander depends on php5-cli; however:
  Package php5-cli is not installed.
 spotcommander depends on php5-sqlite; however:
  Package php5-sqlite is not installed.
 spotcommander depends on php5-curl; however:
  Package php5-curl is not installed.
 spotcommander depends on php5-json; however:
  Package php5-json is not installed.

dpkg: error processing package spotcommander (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 spotcommander
base@music-server:~$
```

Same issue
```
base@music-server:~$ sudo dpkg -i spotcommander-latest.deb
Selecting previously unselected package spotcommander.
(Reading database ... 254900 files and directories currently installed.)
Preparing to unpack spotcommander-latest.deb ...
Unpacking spotcommander (12.4-1) ...
dpkg: dependency problems prevent configuration of spotcommander:
 spotcommander depends on libapache2-mod-php5; however:
  Package libapache2-mod-php5 is not installed.
 spotcommander depends on php5-cli; however:
  Package php5-cli is not installed.
 spotcommander depends on php5-sqlite; however:
  Package php5-sqlite is not installed.
 spotcommander depends on php5-curl; however:
  Package php5-curl is not installed.
 spotcommander depends on php5-json; however:
  Package php5-json is not installed.

dpkg: error processing package spotcommander (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 spotcommander
base@music-server:~$
```

Get the same message when using a GUI. 

And then if I try to install dependancies 

```
base@music-server:~$ sudo apt-get upgrade
Reading package lists... Done
Building dependency tree
Reading state information... Done
You might want to run ‘apt-get -f install’ to correct these.
The following packages have unmet dependencies.
 spotcommander : Depends: libapache2-mod-php5 but it is not installable
                 Depends: php5-cli but it is not installable
                 Depends: php5-sqlite but it is not installable
                 Depends: php5-curl but it is not installable
                 Depends: php5-json but it is not installable
E: Unmet dependencies. Try using -f.
base@music-server:~$ sudo apt-get install -f
Reading package lists... Done
Building dependency tree
Reading state information... Done
Correcting dependencies... Done
The following packages will be REMOVED
  spotcommander
0 to upgrade, 0 to newly install, 1 to remove and 1 not to upgrade.
1 not fully installed or removed.
After this operation, 5,243 kB disk space will be freed.
Do you want to continue? [Y/n]
```

**Comment by olejon** [comment]
Seems like SpotCommander is installed, probably since you used dpkg which does not resolve deps. Run: `dpkg -P spotcommander`. Does it work?

**Comment by changeableface** [comment]
```
base@music-server:~$ sudo dpkg -P spotcommander
(Reading database ... 255283 files and directories currently installed.)
Removing spotcommander (12.4-1) ...
```

**Comment by olejon** [comment]
So now you can do a `sudo apt-get update && sudo apt-get dist-upgrade`without problems?

**Comment by olejon** [comment]
Read above first. Your last comment gives a big clue I think. You had an old version of SpotCommander installed (v 12.4) when v 12.8 is the latest. Try this:

1. `sudo apt-get clean`
2. Reboot
3. `sudo apt-get clean && sudo apt-get --purge autoremove`
4. `sudo apt-get update && sudo apt-get dist-upgrade`
5. Reboot

Try to install SpotCommander the regular way. If not it can be force installed. Keep me updated.

**Comment by olejon** [comment]
Sorry, updated my post above again.

**Comment by olejon** [comment]
Read **above above** post. Please use `sudo gdebi` to install the spotcommander deb, to get more output if errors.

**Comment by changeableface** [comment]
Did as you suggested and we're back in action. Thanks for your outstanding efforts!

**Comment by olejon** [comment]
So SpotCommander works?

**Comment by olejon** [comment]
Please let me know so I can close this, if so.

**Comment by changeableface** [comment]
Yup spotcommander works. Please close. Sorry for the hassle, really appreciate your efforts.

**Comment by olejon** [comment]
Great :-) No problem, I'm always curious when things like this happen. Just gotta' find out what the h*** is happening and fix it you know? Probably would have within a few minutes accessing your computer by SSH. Anyway, asking the right questions and answering the right answers usually leads to the problem being solved :-) Thanks for useful feedback. Luckily Linux users tend to provide useful feedback, and not just "it does not work, this is shit!".

**Comment by olejon** [comment]
Please remember to rate SpotCommander on Google Play if you use an Android device and haven't already done so :-)
